### PR TITLE
kv.proto: docs fix on watch event of DELETE/EXPIRE

### PR DESCRIPTION
**Pull request by hongchaodeng**

### CODE REVIEW

**Comment by xiang90** [comment]
lgtm

File: storage/storagepb/kv.proto [hunk 1]
```diff
@@ -34,6 +34,7 @@ message Event {
   EventType type = 1;
   // A PUT event contains current kv pair.
   // A PUT event with kv.Version=1 indicates the creation of a key.
-  // A DELETE/EXPIRE event contains previous kv pair
+  // A DELETE/EXPIRE event contains previous kv pair's key and
```
Comment (heyitsanthony) [inline storage/storagepb/kv.proto]: A DELETE/EXPIRE event contains the deleted key with its modification revision set to the revision of deletion.

**Comment by gyuho** [comment]
lgtm.

**Comment by hongchaodeng** [comment]
@heyitsanthony Fixed

**Comment by heyitsanthony** [comment]
lgtm thanks

**Pull request merged**